### PR TITLE
license access change

### PR DESCRIPTION
**Pull request by abby-cyber**

### CODE REVIEW

**Review by cooper-lzy — DISMISSED** [review]

File: docs-2.0/nebula-dashboard-ent/2.deploy-connect-dashboard-ent.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Before deploying Dashboard, you must do a check of these:
 
   !!! enterpriseonly
 
-        The license is only available in the Enterprise Edition. To obtain the license, send email to inquiry@vesoft.com.
+        The license is only available in the Enterprise Edition. To obtain the license, submit answers on the [Apply for Nebula Dashboard Free Trial](https://forms.gle/AxVejqKjWT9Z8HLq8) page.
```
Comment (randomJoe211) [inline docs-2.0/nebula-dashboard-ent/2.deploy-connect-dashboard-ent.md:29]: `submit answers` is not necessary.

```suggestion
        The license is only available in the Enterprise Edition. To obtain the license, apply for [Nebula Dashboard Free Trial](https://forms.gle/AxVejqKjWT9Z8HLq8).
```

File: docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md [hunk 2]
```diff
@@ -33,6 +33,11 @@ Before you deploy Explorer, you must do a check of these:
 
 - The Linux distribution is CentOS.
 - GO of version above 1.13 is installed.
+- The license is ready.
+
+  !!! enterpriseonly
+
+        License is only available in the Enterprise Edition. To obtain the license, submit answers on the [Apply for Nebula Explorer Free Trial](https://docs.google.com/forms/d/e/1FAIpQLSctV0HC0HDxZc-65IHj44qpZk3fATBPAYM1SjJsbXmdL5cs9w/viewform) page.
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md:40]:
```suggestion
        License is only available in the Enterprise Edition. To obtain the license, apply for [Nebula Explorer Free Trial](https://docs.google.com/forms/d/e/1FAIpQLSctV0HC0HDxZc-65IHj44qpZk3fATBPAYM1SjJsbXmdL5cs9w/viewform).
```

File: docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md [hunk 3]
```diff
@@ -123,6 +124,11 @@ Before deploying Explorer, you must check the following information:
...
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md:131]:
```suggestion
        License is only available in the Enterprise Edition. To obtain the license, apply for [Nebula Explorer Free Trial](https://docs.google.com/forms/d/e/1FAIpQLSctV0HC0HDxZc-65IHj44qpZk3fATBPAYM1SjJsbXmdL5cs9w/viewform) page.
```

File: docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md [hunk 4]
```diff
@@ -123,6 +124,11 @@ Before deploying Explorer, you must check the following information:
 
 - The Linux distribution is CentOS.
 - GO of version above 1.13 is installed.
+- The license is ready.
+
+  !!! enterpriseonly
+
+        License is only available in the Enterprise Edition. To obtain the license, apply for [Nebula Explorer Free Trial](https://docs.google.com/forms/d/e/1FAIpQLSctV0HC0HDxZc-65IHj44qpZk3fATBPAYM1SjJsbXmdL5cs9w/viewform) page.
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md:131]:
```suggestion
        License is only available in the Enterprise Edition. To obtain the license, apply for [Nebula Explorer Free Trial](https://docs.google.com/forms/d/e/1FAIpQLSctV0HC0HDxZc-65IHj44qpZk3fATBPAYM1SjJsbXmdL5cs9w/viewform).
```

**Review by izhuxiaoqing — APPROVED** [review]

**Review by foesa-yang — APPROVED** [review]

**Pull request merged**